### PR TITLE
Search button shows inputted text and reset button

### PR DESCRIPTION
**Pull request by TurnbulA**
Search will be able to show particular items through key words or searching for particular items

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -138,10 +138,10 @@ form {
 .c-menu-status__text {
   margin-right: 6rem;
 }
-.c-menu-status--hide {
+.u-display--hide {
```
Comment (mali891) [inline style.css]: Do you need both `.u-display--hide` and `.u-display--none` ?

File: style.css [hunk 2]
```diff
@@ -174,9 +175,16 @@ form {
...
   cursor: pointer;
 }
-.c-list-item--hide {
+.u-display--none {
   display: none;
 }
+.u-display--block {
+  display: block;
+}
+.u-display--none-search {
```
Comment (mali891) [inline style.css:178]: Why do you need a separate class for search?

File: todos.js [hunk 3]
```diff
@@ -68,10 +68,10 @@ const filterCompletedItems = () => {
 };
 
 const deleteEvent = () => {
-  const todoListItems = Array.from(
+  const todomainItems = Array.from(
     document.querySelectorAll(".c-list-item__delete")
   );
-  todoListItems.forEach(todoListItem => {
+  todomainItems.forEach(todoListItem => {
```
Comment (mali891) [inline todos.js:72]: todoMainItems

File: todos.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-let listItems;
+let mainItems;
 
 window.addEventListener("DOMContentLoaded", () => {
   fetch("https://jsonplaceholder.typicode.com/todos")
     .then(response => response.json())
     .then(json => {
       renderJson(json);
-      listItems = renderJson(json);
+      mainItems = renderJson(json);
```
Comment (stephen-algeo) [inline todos.js:8]: why are we running renderJson twice?

**Review by mali891 — APPROVED** [review]

**Pull request merged**